api/client/proto/authservice.proto

```diff
@@ -876,8 +894,8 @@ service AuthService {
 
     // GetNode retrieves a node described by the given request.
     rpc GetNode(types.ResourceInNamespaceRequest) returns (types.ServerV2);
-    // GetNodes retrieves all nodes.
```

Comment:
`GetNodes` has been intentionally removed to force http fallback, so this doesn't break backwards compatibility.